northstar/src/runtime/debug.rs

```diff
@@ -131,9 +187,9 @@ impl Strace {
         self.token.cancel();
 
         // Stop strace - if not already existed - irgnore any error
+        let pid = self.child.id().unwrap();
```

Comment:
I would not unwrap() for a log message

Reply:
What log message? 
Getting the pid of a strace child is very likely be good. Will change that to an `expect`